api/src/api/notebooks.ts

```diff
@@ -203,7 +205,11 @@ api.get(
     const metadata = await getNotebookMetadata(project_id);
     const uiSpec = await getNotebookUISpec(project_id);
     if (metadata && uiSpec) {
-      res.json({metadata, 'ui-specification': uiSpec});
+      res.json({
+        metadata,
+        // silly business
+        'ui-specification': uiSpec as unknown as Record<string, unknown>,
```

Comment:
Don't we have a type for uispec? ProjectUIModelDetails. I see your zod type is just a record...

Reply:
Defer - see https://jira.csiro.au/browse/BSS-340